tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "outDir": "build/dist",
     "module": "esnext",
-    "target": "es5",
+    "target": "es6",
```

Comment:
Why did you change that? This change is not desired, we were targeting es5 on purpose.

Reply:
That's my bad. I think I changed this to use some ES6 only code, but I have since removed the code. I will change this back